examples/item_custom_proj.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
the other example files should be updated with these fields, too.

Reply:
Done!